### PR TITLE
🧹 Update dependencies

### PR DESCRIPTION
**Pull request by czunker**
Signed-off-by: Christian Zunker <christian@mondoo.com>

### CODE REVIEW

File: config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml [hunk 1]
```diff
@@ -149,6 +149,25 @@ spec:
                     description: ResourceRequirements describes the compute resource
                       requirements.
                     properties:
+                      claims:
```
Comment (czunker) [inline config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml:152]: This is new in v1.26: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#resourcerequirements-v1-core

Comment (joelddiaz) [reply config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml:152]: Are the test failures due to older versions of k8s not knowing what to do with this new field?

Comment (czunker) [reply config/crd/bases/k8s.mondoo.com_mondooauditconfigs.yaml:152]: Strangely, the same error happened before, when `claims` was missing. I have to dig a bit deeper to find out what's happening here.

**Comment by czunker** [comment]
Reverted the 0.26.0 updates of the k8s dependencies. They aren't supported in the operator-sdk: https://github.com/operator-framework/operator-sdk/blob/master/go.mod#L41

Also reverted the controller-runtime. There is some back and forth in the releases about how to handle the generation which failed our tests: https://github.com/kubernetes-sigs/controller-tools/releases

Perhaps, with the next operator-sdk and controller release, we can upgrade to 0.26.0.

**Review by joelddiaz — APPROVED** [review]

**Pull request merged**